### ISSUE TITLE
The accuracy of model test is poor

### ISSUE THREAD

**Issue by aoji0606** [BAAI-DCAI/Bunny#78]
Thanks for your work.

I use the siglip and llama3 for pretrain and fine-tune，but the test results were poor，here are the results：
![image](https://github.com/BAAI-DCAI/Bunny/assets/55875798/db77698e-83a1-448a-93bd-d7b5df349ae5)
Why is that?

**Comment by Isaachhh** [comment]
We use a better strategy to train Phi-3-Mini-based and Llama-3-8B-based Bunny, which would be open-sourced soon.

**Comment by aoji0606** [comment]
Yes, I use Bunny-pretrain-laion-2m for pre-train, and Bunny-695k for fine-tune

**Comment by Isaachhh** [comment]
What's your pre-training learning rate?

**Comment by aoji0606** [comment]
1e-3 for pre-train
2e-5 for fine-tune

**Comment by Isaachhh** [comment]
Do you use lora or full fine-tuning?

**Comment by Isaachhh** [comment]
LoRA would convey a better performance.

**Comment by aoji0606** [comment]
ahhh, ok, thank you

**Comment by aoji0606** [comment]
hi~
What the learning rate should be set when using lora to fine-tune the llama3 model

**Comment by Isaachhh** [comment]
As default, 2e-4 and 2e-5.

**Comment by aoji0606** [comment]
ok thank u

**Comment by Isaachhh** [comment]
Close the issue for now if there's no further discussions. Feel free to reopen it if there's any other questions.
